myApp.js

```diff
@@ -48,13 +48,13 @@ var app = express();
 
 
 module.exports = app;
-var api = require('./server.js');
+const api = require('./server.js');
 app.use(express.static('public'));
 app.disable('strict-transport-security');
 app.use('/_api', api);
 app.get("/", function (request, response) {
   response.sendFile(__dirname + '/views/index.html');
 });
-var listener = app.listen(process.env.PORT || 3000, function () {
-  console.log('Your app is listening on port ' + listener.address().port);
+app.listen(process.env.PORT || 3000, () => {
+  console.log(`Your app is listening on port ${process.env.PORT || 3000}`);
```

Comment:
```suggestion
app.listen(3000, () => {
  console.log('Your app is listening on port 3000');
```
If touching this part I think it'd be better to remove dynamic port altogether, per https://github.com/freeCodeCamp/boilerplate-project-timestamp/pull/51#issuecomment-776940244

Reply:
All right, but what about dotenv file configuration then? And we cannot forget about the fact that these projects suppose to teach people how to create and deploy their future projects in real environments, where you very often have to use the "dynamic" port from the hosting provider.

Reply:
@KowalewskiPawel, the main idea is: If a Camper can/does setup their app on an environment like Heroku/DO, then they will know how to change the port logic. That is, we cannot write logic to support all use-cases, but we can write something to get most started.

IMO, this is far enough in the curriculum where this much boilerplate is not needed, but then it could distract from the aim of the lessons.

Reply:
There's explicit challenge about `.env` files in Basic Node and Express section from APIs and Microservices certification, and at least some in projects from Quality Assurance certification are requiring setting up environment variables for testing. It's not like this dynamic part is giving additional, active, practice with the `.env` files :slightly_smiling_face:

Reply:
And that is the point if we started learning according to a given convention, why should we skip those parts in more advanced projects? Another thing is hardcoding the value that may be changed.

Reply:
Simplest answer - it's out of the scope of the project itself and might not be needed. If somebody will want or need to expand it, that's fine, there are other challenges/projects to help with that.

Reply:
Well, this functionality was there before, so I haven't added it just suggested few things to make code look cleaner and easier to understand. However, if most of you agree that it is time to erase this small detail from the code I am OK with that.

Reply:
No, no, in no way this is about blaming for it being there. Thank you for understanding.

Reply:
So, do you mean that we should get rid of that part completely and my commit was just an occasion for that?